front/extension/config/webpack.ts

```diff
@@ -90,9 +108,15 @@ export const getConfig = async ({
         name: `DustExt [${env}]`,
         color: "#3B82F6",
       }),
+      new webpack.ProvidePlugin({
+        Buffer: ["buffer", "Buffer"],
+      }),
       new webpack.EnvironmentPlugin({
         VERSION: version,
       }),
+      new webpack.DefinePlugin({
+        "process.env.USE_PUBLIC_API": '"http://localhost:3000/api/v1"',
```

Comment:
Should put that in .env.development instead?

Reply:
yes, in that case we can move that to EnvironmentPlugin

Reply:
actually directly handled by DotEnv plugin